src/sentry/relay/config/__init__.py

```diff
@@ -62,7 +62,6 @@
     "organizations:standalone-span-ingestion",
     "organizations:transaction-name-mark-scrubbed-as-sanitized",
     "organizations:transaction-name-normalize",
-    "organizations:user-feedback-ingest",
```

Comment:
Is the Relay PR for this opened yet? We need to make sure that we don't accidentally drop user feedback in external Relays (i.e. internal Relays will have to hard-code this flag when propagating project config downstream).

Reply:
Just opened it!https://github.com/getsentry/relay/pull/4076

Reply:
> We need to make sure that we don't accidentally drop user feedback in external Relays (i.e. internal Relays will have to hard-code this flag when propagating project config downstream).

Sorry I don't think I understand - under what conditions would this happen?

Reply:
@aliu39 customers that run their own Relay instances do not have the updated Relay code yet, so they will still check the feature flag and drop data if it's missing.

I pushed a commit to your Relay PR to prevent that, hope that clarifies what I mean!